src/app/issue-details/issue-details.component.html

```diff
@@ -0,0 +1,50 @@
+<div class="card">
+  <ul class="list-group list-group-flush">
+    <li class="list-group-item">
+      <h6>Assignees</h6>
+      <ul>
+        <li *ngFor="let assignee of issue.assignees">
+          <span>{{assignee}}</span>
+          <span class="float-right">
+            <a class="deleteButton" (click)="unassign(assignee)"><fa-icon [icon]="faMinus" size="xs"></fa-icon></a>
+          </span>
+        </li>
+      </ul>
+      <form [formGroup]="assignForm">
+        <div class="input-group input-group-sm form-group">
+          <input type="text" class="form-control" placeholder="Assign someone" formControlName="text">
+          <div class="input-group-append">
+            <button type="submit" class="btn btn-outline-secondary" (click)="assign()">
+              <fa-icon [icon]="faPlus" size="xs"></fa-icon>
+            </button>
+          </div>
+        </div>
+      </form>
+    </li>
+    <li class="list-group-item">
+      <h6>Labels</h6>
+      <div *ngFor="let label of issue.labels">
+        <span class="badge badge-info text-left mb-1">{{label}}</span>
+        <span class="float-right">
+          <a class="deleteButton" (click)="unLabel(label)"><fa-icon [icon]="faTrash" size="xs"></fa-icon></a>
```

Comment:
Bootstrap has classes that turn links into buttons. Also comes with a red one too.

Reply:
I'm aware of the 'btn' classes but the buttons are too big even with 'btn-xs'. What can I do to make them smaller?

Reply:
That's fair. Then, don't worry. The way to do it would be messing with the padding and margin via the p and m utility classes